tests/ci/sync-to-aws.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -euo pipefail
+
+source /tmp/.env
+
+[ -z "$IMAGE_KEY" ] && error "Could not find '$IMAGE_KEY', qcow image has not been created"
+DOWNLOAD_DIRECTORY="/var/lib/libvirt/images"
+S3_BUCKET_NAME="fedora-testing-farm-image-import"
+AWS_CLI="aws"
+
+# Configure aws region and output format
+$AWS_CLI configure set default.region us-east-2
+$AWS_CLI configure set default.output json
+
+# Remove stop and remove the VM, so qcow image will be available for convertion
+virsh destroy "${IMAGE_KEY}" || true
+virsh undefine "${IMAGE_KEY}" --nvram
+
+# Convert qcow2 image to raw image
+echo "[+] Converting from qcow2 to raw format"
+qemu-img convert -p ${DOWNLOAD_DIRECTORY}/${IMAGE_KEY}.qcow2 ${DOWNLOAD_DIRECTORY}/${IMAGE_KEY}.raw
+
+# Upload the raw and qcow2 images to s3 bucket
+echo "[+] Uploading raw image to 's3://${S3_BUCKET_NAME}/${IMAGE_KEY}.raw"
+$AWS_CLI s3 cp ${DOWNLOAD_DIRECTORY}/${IMAGE_KEY}.raw s3://${S3_BUCKET_NAME}
+$AWS_CLI s3 cp ${DOWNLOAD_DIRECTORY}/${IMAGE_KEY}.qcow2 s3://${S3_BUCKET_NAME}
+
+# Remove the downloaded files
+echo "[+] Removing downloaded and converted image"
+rm -f ${DOWNLOAD_DIRECTORY}/${IMAGE_KEY}.{qcow2,raw}
+
+# Import snapshot
+echo "[+] Import snapshot to EC2"
+IMPORT_SNAPSHOT_ID=$($AWS_CLI ec2 import-snapshot --disk-container Format=raw,UserBucket="{S3Bucket=$S3_BUCKET_NAME,S3Key=${IMAGE_KEY}.raw}" | jq -r .ImportTaskId)
+
+# Wait for image snapshot import
+echo "[+] Waiting for snapshot import"
+until [ "$status" == "completed" ]; do
+    status=$($AWS_CLI ec2 describe-import-snapshot-tasks --import-task-ids $IMPORT_SNAPSHOT_ID | jq -r .ImportSnapshotTasks[0].SnapshotTaskDetail.Status)
+    echo -n "."
+done
+echo
```

Comment:
What is the purpose for using a naked echo? Is it for generating a newline? If that is the case then use the "\n" in the following echo command.

Reply:
Done